ietf/templates/doc/document_top.html

```diff
@@ -9,11 +9,13 @@ <h1>
 </h1>
 <ul class="nav nav-tabs my-3">
     {% for name, t, url, active, tooltip in tabs %}
-        <li {% if tooltip %}title="{{ tooltip }}"{% endif %} class="nav-item">
-            <a class="nav-link {% if t == selected %}active{% endif %}{% if not active %}disabled{% endif %}"
-               {% if active %}href="{{ url }}"{% endif %}>
-                {{ name|capfirst_allcaps }}
-            </a>
-        </li>
+        {% if doc.stream.name|lower == 'ietf' or t not in 'ballot,writeup' %}
```

Comment:
From a maintenance point of view it would be better to not put these tabs in the `tabs` list in the first place in the view code.

Reply:
This will also bury old ballots (from the IESG Tracker days) on non-ietf-stream documents, when the conflict review was an IESG ballot on the doc instead of an IESG ballot on a conflict-review document.

Reply:
I think research will be needed to see what will work - maybe we place the tabs if the stream is ietf _or_ there is an IESG ballot exists. And I wonder if we can treat the two tabs differently.

Reply:
See. for example, https://datatracker.ietf.org/doc/rfc5345/ballot/